sotodlib/coords/pointing_model.py

```diff
@@ -167,20 +171,25 @@ def model_sat_v1(params, az, el, roll):
             raise ValueError(f'Handling of model param "{k}" is not implemented.')
 
     # Construct offsetted encoders.
-    az = az + params['enc_offset_az'] * DEG
-    el = el + params['enc_offset_el'] * DEG
-    roll = roll - params['enc_offset_boresight'] * DEG
+    az_twist = params['az_rot'] * (el + params['enc_offset_el'])
+    az = az + params['enc_offset_az'] + az_twist
+    el = el + params['enc_offset_el'] 
+    roll = roll - params['enc_offset_boresight'] 
 
     # Rotation that tilts the base (referred to vals after enc correction).
     base_tilt = get_base_tilt_q(params['base_tilt_cos'], params['base_tilt_sin'])
 
     # Rotation that takes a vector in array-centered focal plane coords
     # to a vector in boresight-rotation-centered focal plane coords.
-    q_fp_bs = ~quat.rotation_xieta(params['bs_xi0'], params['bs_eta0'])
+    q_fp_rot = ~quat.rotation_xieta(params['fp_rot_xi0'], params['fp_rot_eta0'])
+    
+    # Rotation that moves the center of the focal plane to fp_offset_(xi, eta)0.  
+    q_fp_offset = quat.rotation_xieta(params['fp_offset_xi0'], params['fp_offset_eta0'])
+
 
     # Horizon coordinates.
     q_hs = (base_tilt * quat.rotation_lonlat(-az, el)
-            * ~q_fp_bs * quat.euler(2, roll) * q_fp_bs)
+             *q_fp_offset * ~q_fp_rot* quat.euler(2, roll) * q_fp_rot)
```

Comment:
add space after first `*` (and align that `*` under the `b` of `base_tilt`).

Add space after `~q_fp_rot`, before `*`.

Reply:
done